template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Views/Account/Login.cshtml

```diff
@@ -59,7 +60,8 @@
             <h4>Use another service to log in.</h4>
             <hr />
             @{
-                var loginProviders = SignInManager.GetExternalAuthenticationSchemes().ToList();
+                var schemes = await SchemeProvider.GetAllSchemesAsync();
```

Comment:
Not really, you underestimate how many places I had to do this exact code pattern last night :)